friends/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "friends",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
+    "react-scripts": "2.0.5"
```

Comment:
you should have axios in your package.json (should have been added by `yarn add axios`